### PR TITLE
Not nord esque

### PR DESCRIPTION
**Pull request by f3fora**
I wanted to add some plugins and small corrections. I end up rewriting about everything.
Now the theme is Nord and not nord-esque.

If you like it, merge this PR, otherwise I will create an independent color-scheme,

### CODE REVIEW

**Comment by maaslalani** [comment]
Hey! This looks solid, thanks so much for contributing. I will double check it works and then merge

File: lua/nordbuddy.lua [hunk 1]
```diff
@@ -1,341 +1,498 @@
...
+        {'DiffAdd', c.nord14, c.nord1}, --
+        {'DiffChange', c.nord13, c.nord1}, --
+        {'DiffDelete', c.nord11, c.nord1}, --
+        {'DiffText', c.nord11, c.nord1}, --
+        {'GitGutterAdd', c.nord14, c.nord1}, --
+        {'GitGutterChange', c.nord13, c.nord1}, --
+        {'GitGutterDelete', c.nord11, c.nord1}, --
+        {'NeogitDiffAddHighlight', c.nord14, c.nord1}, --
+        {'NeogitDiffDeleteHighlight', c.nord11, c.nord1}, --
+        {'NeogitDiffContextHighlight', c.nord13, c.nord1}, --
```
Comment (maaslalani) [inline lua/nordbuddy.lua]:
```suggestion
        {'NeogitDiffContextHighlight', c.nord13, c.nord0}, --
```

File: lua/nordbuddy.lua [hunk 2]
```diff
@@ -1,341 +1,498 @@
...
+        {fields, c.nord9, cno, i}, --
+        {functions, c.nord4, cno, i}, --
+        {includes, c.nord10}, --
+        {keywords, c.nord9, cno, sno}, --
+        {labels, c.nord10, cno, i}, --
+        {namespaces, c.nord4, cno, i}, --
+        {nones, c.nord4}, --
+        {operators, c.nord9}, --
+        {parameters, c.nord4, cno, i}, --
+        {properties, c.nord10}, --
```
Comment (maaslalani) [inline lua/nordbuddy.lua]:
```suggestion
        {properties, c.nord9}, --
```

File: lua/nordbuddy.lua [hunk 3]
```diff
@@ -1,341 +1,498 @@
...
+        {punctuations, c.nord9}, --
+        {repeats, c.nord10, cno, i}, --
```
Comment (maaslalani) [inline lua/nordbuddy.lua]: Repeats is nord9 in the `nord-vim` colorscheme: https://github.com/arcticicestudio/nord-vim/blob/develop/colors/nord.vim#L291

```suggestion
        {repeats, c.nord9, cno, i}, --
```

**Review by maaslalani — APPROVED** [review]
Looks great! Few suggestions

File: lua/nordbuddy.lua (hunk 2 above)
Comment (maaslalani) [reply lua/nordbuddy.lua]: They use `nord9` here: https://github.com/arcticicestudio/nord-vim/pull/253/files#diff-4a2da78e9170e9cba996ab45b000ac6bce8d74246cba502fae9cb36c8c43ed24R821

**Comment by f3fora** [comment]
Thanks! I actually solved the issue described in the README. However, I forgot to removed it. There's some inconsistency for example in `vim` file without treesitter.

**Pull request merged**